src/main/java/core/basesyntax/operations/BalanceOperation.java

```diff
@@ -0,0 +1,13 @@
+package core.basesyntax.operations;
+
+import core.basesyntax.FruitTransaction;
+import java.util.Map;
+
+public class BalanceOperation implements OperationHandler {
+    @Override
+    public Map<String, Integer> getCalculation(
+            Map<String, Integer> fruits, FruitTransaction fruitTransaction) {

```

Comment:
```suggestion
    public void apply(FruitTransaction fruitTransaction) {
    Storage.storage.put(....
```
Fix for other handlers